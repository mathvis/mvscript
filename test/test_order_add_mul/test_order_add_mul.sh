#!/bin/bash
helix test_order_add_mul.mvs test_order_add_mul.out --vsplit
