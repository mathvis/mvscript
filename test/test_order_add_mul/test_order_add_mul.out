Expr (Type (Int 12))
